app/src/main/java/io/github/polysmee/appointments/fragments/DataPasser.java

```diff
@@ -0,0 +1,7 @@
+package io.github.polysmee.appointments.fragments;
+
+import java.util.Set;
+
+public interface DataPasser {
```

Comment:
hehe what is that

Reply:
Ups, forgot the Javadoc. I fix this within the hour